config/vufind/LibGuidesAZ.ini

```diff
@@ -0,0 +1,16 @@
+; Settings for the LibGuides AZ Databases List are likely to be the same as those 
+; for the canonical "guides" part of LibGuides, defined in LibGuides.ini.
+; But override here if needed.
+[Parent_Config]
+relative_path = ./LibGuides.ini
+
+; ---------- config.ini settings ----------
+
+;[General]
+;timeout = 30
+;iid = my-id
+;version = 2
+;baseUrl = https://lgapi.libapps.com/widgets.php
+;default_limit        = 20
+;limit_options        = 10,20,40,60,80,100
+;defaultSearch = "general"
```

Comment:
You might want to add an end-of-file line break here for consistency with most other files in the project.

Reply:
Done!